src/shared/gpt.c

```diff
@@ -158,6 +158,23 @@ static const char *const partition_designator_table[] = {
 
 DEFINE_STRING_TABLE_LOOKUP(partition_designator, PartitionDesignator);
 
+static const char *const partition_mountpoint_table[] = {
```

Comment:
please size this explicitly to _PARTITION_DESIGNATOR_MAX so that we can be sure that there's a NULL entry allocated for unlisted designators, no matter what.

(or to say this differently, right now if I call `gpt_partition_type_mountpoint_nulstr(PARTITION_LINUX_GENERIC)` I'll get uninitialized memory back.)